addons/event_sale/models/event_event.py

```diff
@@ -70,6 +70,7 @@ def _compute_start_sale_date(self):
     @api.depends('sale_order_lines_ids')
     def _compute_sale_total_price(self):
         for event in self:
+            event.currency_id = self.env.company.currency_id.id
```

Comment:
Calling on to our mighty @tde-banana-odoo here.
Is this one of those times where we should use `@api.depends_context('company')` ?

Reply:
@awa-odoo 
@tde-banana-odoo 
The event belongs to a company (we can not read the events of an other company)

So, it should be ok no ?

Reply:
Why not simply converting into the event's company ?

Otherwise indeed depends company.

Moreover while you are at it triggers should be more precise for that method.

Reply:
@tde-banana-odoo 
Indeed, best solution :)

If we do not have more comment on this task, we can move forward :) (as it block event social)

Reply:
Don't be hasty little hobbit.